### DISCUSSION TITLE
Open source software could be used to create a PKI in diferents instances of EJBCA ?

### DISCUSSION THREAD

**Discussion by bsjaramillo** [Keyfactor/ejbca-ce#292] [General]
I mean, If I want to create a RootCA and others CAs in other servers.

**Comment by primetomas** [comment]
Yes. See here: https://doc.primekey.com/ejbca/ejbca-operations/ejbca-operations-guide/ca-operations-guide/managing-cas

**Comment by bsjaramillo** [comment]
I have some questions about this type of installation, in the context of, for example, implementing an electronic signature service in a company that has other branches. I would greatly appreciate your help.

1. What would be the most appropriate approach regarding the management of databases, managing a database installed in the root node or in each instance of EJBCA?

2. How are the certificates generated by other instances of EJBCA (SubCAs) related to the root (RootCA) of the company or other branches (SubCAs)? In other words, how is that certificate validated or verified in any node of the company?

Thank you very much in advance.
